packages/embed/src/utils/get-transferable-query-params.ts

```diff
@@ -0,0 +1,18 @@
+export const getTransferableQueryParams = (queryStrings: QueryStrings, transferableUrlParameters?: string[]) => {
```

Comment:
This is not "query string", but rather "query params".
```suggestion
export const getTransferableQueryParams = (queryParams: Record<string, any>, transferableUrlParameters?: string[]) => {
```